bin/playwright.sh

```diff
@@ -9,8 +9,8 @@ export PLAYWRIGHT_ARGS=$@
 export PLAYWRIGHT_VERSION=$(version)
 export TEST_COMMAND=${TEST_COMMAND:-test:playwright:local}
 
-echo Running Playwright v$PLAYWRIGHT_VERSION as $USER_ID with Playwright arguments $PLAYWRIGHT_ARGS
+cp -r test/locales/* src/locales
```

Comment:
I wonder if it's worth doing this in the Dockerfile.playwright instead of in here.

Reply:
I removed it because we actually need the matching locale file in other tests, not only the Playwright ones.
When the `valid-locales.json` is not copied in setup-env step in other tests such as Storybook, the app fails to build. Also, simply copying the `valid-locales.json` is not enough because the app requires all of the translation `json` listed in `valid-locales.json` to build.